flashinfer/quantization/kernels/nvfp4_quantize.py

```
@@ -1877,6 +1930,178 @@ def nvfp4_quantize_cute_dsl(
     return fp4_output, scale_output
 
 
+@flashinfer_api
+def silu_and_mul_nvfp4_quantize_cute_dsl(
+    input: torch.Tensor,
```

Ruff (0.15.21)

[error] Function argument `input` is shadowing a Python builtin

(A002)